packages/did-ethr-resolver/src/utils/prototypes.ts

```diff
@@ -12,7 +11,3 @@ export function withKey(
   });
   return signer as IdentityOwner;
 }
```

Comment:
This can be removed now, I believe. At any rate, we don't want this function to be used publicly as it still has ethers API